data_dir/cs_lwt_perf_big.yaml

```diff
@@ -0,0 +1,107 @@
+# Keyspace Name
+keyspace: cs_lwt_perf_big
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE cs_lwt_perf_big WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: targettable
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE targettable (
+        pr1 timeuuid,
+        cl1 text,
+        aux1 timeuuid,
+        aux2 int,
+        aux3 smallint,
+        aux4 bigint,
+        aux5 double,
+        aux6 float,
+        aux7 tinyint,
+        aux8 decimal,
+        aux9 text,
+        PRIMARY KEY(pr1, cl1)
+  ) WITH compaction = { 'class':'LeveledCompactionStrategy' }
+    AND comment='A test table for cs_lwt_perf_big'
+
+### Column Distribution Specifications ###
+
+columnspec:
+  - name: pr1
+    size: uniform(1..1k)
+    population: uniform(1..20M)
+  - name: cl1
+    size: uniform(1..1k)
+    population: uniform(1..20M)
```

Comment:
Did you intend to have this as clustering key?
if so, you need to add "cluster:"